modules/installation-vsphere-machines.adoc

```diff
@@ -165,9 +165,13 @@ $ export IPCFG="ip=192.168.100.101::192.168.100.254:255.255.255.0:::none nameser
 $ govc vm.change -vm "<vm_name>" -e "guestinfo.afterburn.initrd.network-kargs=${IPCFG}"
 ----
 +
+[comment]
+--
+//Need to replace with alternative text to give a possible use case of diagnosing cluster performance issues. Possible a resource pool configuration.
```

Comment:
Query for SME: VMC docs exist for 4.10 to 4.13, so I should remove any Latency Sensitivity references from these docs also?

Reply:
Yeah I would think so